### PR TITLE
[TASK] Set default dir when saving txn manually (DC-184)

### PR DESCRIPTION
**Pull request by mrajvanshy**

### CODE REVIEW

File: src/js/tabs/settingsgateway.js [hunk 1]
```diff
@@ -145,9 +138,7 @@ SettingsGatewayTab.prototype.angular = function(module)
         $scope.hash = tx.hash('HASH_TX_ID', false, undefined);
         $scope.offlineSettingsChange = true;
         $scope.edit[type] = false;
-        if ($scope.userBlob.data.defaultDirectory) {
```
Comment (MatthewPhinney) [inline src/js/tabs/settingsgateway.js]: Do you want this change in `removeFlag` too?

Comment (mrajvanshy) [inline src/js/tabs/settingsgateway.js]: @MatthewPhinney fixed!

File: src/js/controllers/app.js [hunk 2]
```diff
@@ -59,6 +64,40 @@ module.controller('AppCtrl', ['$rootScope', '$compile', 'rpId', 'rpNetwork',
...
 
+  // Set default directory if it has not already been set
+  $scope.fileInputClick = function(txnName, txData) {
+    fileDialog.openDir(function(evt) {
+      $scope.$apply(function() {
+        $scope.defaultDirectory = evt;
+        $scope.$watch('userBlob', function() {
+          if ($scope.userBlob.data && $scope.userCredentials.username) {
+            $scope.userBlob.set('/defaultDirectory', evt);
+            $scope.defaultDirectory = $scope.userBlob.data.defaultDirectory;
```
Comment (MatthewPhinney) [inline src/js/controllers/app.js]: It looks like `$scope.defaultDirectory` is already set on line 71. Do we need this line?

File: src/js/tabs/coldwalletsettings.js [hunk 3]
```diff
@@ -24,23 +24,6 @@ ColdwalletSettingsTab.prototype.angular = function(module) {
     if (!id.loginStatus) {
       id.goId();
     }
-    if ($scope.userBlob.data.defaultDirectory) {
```
Comment (MatthewPhinney) [inline src/js/tabs/coldwalletsettings.js]: Is the plan here to remove the buttons from the UI so the user can't change the default directory?

Comment (mrajvanshy) [inline src/js/tabs/coldwalletsettings.js]: I think the user should still be able to change the default directory from this page if they want to later on, but the function is moved to app.js

Comment (MatthewPhinney) [inline src/js/tabs/coldwalletsettings.js]: Makes sense!

**Comment by MatthewPhinney** [comment]
LGTM

**Pull request merged**